addons/binding/org.openhab.binding.lutron/ESH-INF/thing/thing-types.xml

```diff
@@ -167,6 +167,154 @@
+                <default>30</default>
+            </parameter>            
+            <parameter name="shadeZones" type="text">
+                <label>Shade Zones</label>
+                <description>Comma separated list of zones that are shades</description>
+                <default></default>
+            </parameter>            
+        </config-description>
+    </bridge-type>
+
+    <thing-type id="grafikeye">
+        <supported-bridge-type-refs>
+            <bridge-type-ref id="prgbridge"/>
+        </supported-bridge-type-refs>
+
+        <label>Grafik Eye</label>
+        <description>Controls a Grafik Eye</description>
+
+        <channels>
+            <channel id="scene" typeId="scene"/>
+            <channel id="scenelock" typeId="scenelock"/>
+            <channel id="sceneseq" typeId="sceneseq"/>
+            <channel id="zonelock" typeId="zonelock"/>
+            <channel id="zonelower1" typeId="zonerampdown"/>
+            <channel id="zonelower2" typeId="zonerampdown"/>
+            <channel id="zonelower3" typeId="zonerampdown"/>
+            <channel id="zonelower4" typeId="zonerampdown"/>
+            <channel id="zonelower5" typeId="zonerampdown"/>
+            <channel id="zonelower6" typeId="zonerampdown"/>
+            <channel id="zonelower7" typeId="zonerampdown"/>
+            <channel id="zonelower8" typeId="zonerampdown"/>
+            <channel id="zoneraise1" typeId="zonerampup"/>
+            <channel id="zoneraise2" typeId="zonerampup"/>
+            <channel id="zoneraise3" typeId="zonerampup"/>
+            <channel id="zoneraise4" typeId="zonerampup"/>
+            <channel id="zoneraise5" typeId="zonerampup"/>
+            <channel id="zoneraise6" typeId="zonerampup"/>
+            <channel id="zoneraise7" typeId="zonerampup"/>
+            <channel id="zoneraise8" typeId="zonerampup"/>
+            <channel id="zonefade" typeId="zonefade"/>
+            <channel id="zoneintensity1" typeId="zoneintensity"/>
+            <channel id="zoneintensity2" typeId="zoneintensity"/>
+            <channel id="zoneintensity3" typeId="zoneintensity"/>
+            <channel id="zoneintensity4" typeId="zoneintensity"/>
+            <channel id="zoneintensity5" typeId="zoneintensity"/>
+            <channel id="zoneintensity6" typeId="zoneintensity"/>
+            <channel id="zoneintensity7" typeId="zoneintensity"/>
+            <channel id="zoneintensity8" typeId="zoneintensity"/>
+            <channel id="zoneshade1" typeId="zoneshade"/>
+            <channel id="zoneshade2" typeId="zoneshade"/>
+            <channel id="zoneshade3" typeId="zoneshade"/>
+            <channel id="zoneshade4" typeId="zoneshade"/>
+            <channel id="zoneshade5" typeId="zoneshade"/>
+            <channel id="zoneshade6" typeId="zoneshade"/>
+            <channel id="zoneshade7" typeId="zoneshade"/>
+            <channel id="zoneshade8" typeId="zoneshade"/>
```

Comment:
I believe you need labels on these in order to differentiate channels of the same type in the UI.